### DISCUSSION TITLE
不支持sum() over(order by range between) 动态时间窗口函数

### DISCUSSION THREAD

**Discussion by colagy** [apache/doris#44670] [Ideas]
目前doris支持了rows between 没有支持range between 很多指标无法实现. 我们是从greenplum迁移到doris 如果实现不了range between 无法迁移.
sum(计数) over(partition by 支持时间 order by 销售主单号 range between interval '31 days' preceding ans interval '1 day' following)

**Comment by echo-hhj** [comment]
这个暂时不支持，后续会支持

**Reply by colagy** [reply]
你好  请问什么时候可以实现呢

**Reply by colagy** [reply]
你好  请问什么时候可以实现呢

**Reply by echo-hhj** [reply]
这个目前优先级不高，没有确定的时间
